atomata/capstone_WebClient
# Apparatus trigger list

### Summary 
This PR adds the action list for each selected apparatus. When the user selects an apparatus from the appratus box , the acion list show the list of available actions. the user can add these actions to the action sequence box by clicking on the + button.
### Details
- The user can select an apparatus compnent and see the available actions [ only void actions]
- The user can click on + next to each action to add it to the sequence box
- the user can remove an action from the sequence box by clicking on the bin 
- 
### How to test?
- run the app
- select an appratus from the selection menu
- you should see the appratus component in the appratus box
- Click on any of the appratuses, if they support an action, you should see them in the action list. [only wobble sphere supports action for now] 
- Click on the + and add as many actions to the action sequence box
- you can remove any number of actions from the action sequence box
### Checks 
- [ X] Tested Changes
